src/PhpCsFixer/Rules.php

```diff
@@ -226,6 +226,13 @@ private static function getBaseRules(): array
                 'identical' => false,
                 'less_and_greater' => false,
             ],
+            'trailing_comma_in_multiline' => [
+                'after_heredoc' => true,
+                'elements' => [
+                    'array_destructuring',
+                    'arrays',
+                ],
```

Comment:
Will this block trailing commas in method calls?

Reply:
According to https://cs.symfony.com/doc/rules/control_structure/trailing_comma_in_multiline.html, I believe this should be our config (which matches `PER-CS2.0`)

```suggestion
                'elements' => ['arguments', 'array_destructuring', 'arrays', 'match', 'parameters']
```

Reply:
Fixed